crates/red_knot_python_semantic/src/types.rs

```diff
@@ -1834,31 +1925,30 @@ impl<'db> ClassType<'db> {
     /// Returns the class member of this class named `name`.
     ///
     /// The member resolves to a member of the class itself or any of its bases.
-    pub fn class_member(self, db: &'db dyn Db, name: &str) -> Type<'db> {
+    pub fn class_member(self, db: &'db dyn Db, name: &str) -> SymbolLookupResult<'db> {
```

Comment:
My use of the term symbol has been imprecise in the past and @carljm corrected me many times. do all `name`s resolve to a symbol? If not, should we rename the `SymbolLookupResult` type?